config.js.template

```diff
@@ -0,0 +1,8 @@
+module.exports = {
```

Comment:
Add comments about each variable or add to README

Reply:
Done. Thanks.